.github/workflows/hello_world_multiplatform.yaml

```diff
@@ -29,6 +29,12 @@ jobs:
         os: [ubuntu-22.04, ubuntu-24.04, macos-13, macos-14, windows-2022]
     runs-on: ${{ matrix.os }}
     steps:
+      - name: Install packages
+        if: startsWith(runner.os, 'Linux')
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y g++-multilib
```

Comment:
should this go straight in `https://github.com/zephyrproject-rtos/action-zephyr-setup/blob/main/action.yml#L48`? May save some hassle for c++ user for not much inconvenience for everyone else

Reply:
Probably, yes. I only added it here to be able to verify the builds. We can add it to the setup action and then remove it from here when bumping the version of that action?

Reply:
If you open a PR for that I can merge it and point the v1 tag to it right now, no need to bump it.

Reply:
I've opened https://github.com/zephyrproject-rtos/action-zephyr-setup/pull/19